### PR TITLE
doc(utils): round out JSDocs in `src/utils/helpers.ts`

### PR DESCRIPTION
**Pull request by alicewriteswrongs**
This rounds out the JSDoc comments in the `helpers` file, which was a bit under-documented before.

## Pull request checklist

Please check if your PR fulfills the following requirements:
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [ ] Build (`npm run build`) was run locally and any changes were pushed
- [ ] Unit tests (`npm test`) were run locally and passed
- [ ] E2E Tests (`npm run test.karma.prod`) were run locally and passed
- [ ] Prettier (`npm run prettier`) was run locally and passed

## Pull request type

 

 

Please check the type of change your PR introduces:
- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?

GitHub Issue Number: N/A

## What is the new behavior?

Just a JSDoc change

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: src/utils/helpers.ts [hunk 1]
```diff
@@ -39,19 +39,46 @@ export const toCamelCase = (str: string) => {
   return pascalCase.charAt(0).toLowerCase() + pascalCase.slice(1);
 };
 
-export const toTitleCase = (str: string) => str.charAt(0).toUpperCase() + str.slice(1);
+/**
+ * Capitalize the first letter of a string
+ *
+ * `toTitleCase('foo') => 'Foo'
+ *
```
Comment (rwaskiewicz) [inline src/utils/helpers.ts:46]: Minor, the first '`' is unmatched.

We can probably rm this just as easily, I'd expect the first sentence of the JSDoc does it justice
```suggestion
```

File: src/utils/helpers.ts [hunk 2]
```diff
@@ -86,14 +132,31 @@ export const unique = <T>(array: T[], predicate: (item: T) => any = (i) => i): T
   });
 };
 
+/**
+ * A utility for building an object from an interable very similar to
```
Comment (rwaskiewicz) [inline src/utils/helpers.ts:136]:
```suggestion
 * A utility for building an object from an iterable very similar to
```

**Review by rwaskiewicz — APPROVED** [review]
Two non-blocking comments

File: src/utils/helpers.ts (hunk 1 above)
Comment (alicewriteswrongs) [reply src/utils/helpers.ts:46]: makes sense! I guess 'this capitalizes things' isn't too ambiguous haha

**Review by tanner-reits — APPROVED** [review]

**Pull request merged**